### ISSUE TITLE
Version not reported (and no help given) if bad/no arguments given

### ISSUE THREAD

**Issue by Fish-Git** [SDL-Hercules-390/hyperion#540]
Hercules should really support _either:_ 

* a) always reporting its version even when no (or invalid) arguments are given, or
* b) a new `--version` argument to report its current version then exit.

_**Additionally**_, it should also report all of the valid arguments that it supports too, just like virtually all command line programs out there do! _(Hercules currently doesn't even do that! Sheesh!)_

&nbsp;
All of the Hercules utilities, for example, always report right away what version they are even when no arguments are given:

<pre>
C:\hercules> dasdser
<b>HHC02499I Hercules utility dasdser - DASD serial number utility - version 4.6.0.10880-SDL-DEV-g468b76cd</b>
HHC01414I (C) Copyright 1999-2023 by Roger Bowler, Jan Jaeger, and others
HHC01417I ** The SoftDevLabs version of Hercules **
HHC01415I Build date: Jan 24 2023 at 10:49:22
HHC03100I Usage: dasdser  image  [serial]
HHC03100I   image    base CKD dasd image file
HHC03100I   serial   serial number to set. If not given then the
HHC03100I            existing serial number is simply displayed.
</pre>

But Hercules itself however, unfortunately does not:

```
C:\hercules> hercules
HHC00100I Thread id 00002380, prio 5, name 'impl_thread' started
HHC00100I Thread id 000011e4, prio 4, name 'logger_thread' started
HHC02342S Configuration file 'hercules.cnf' not found: No such file or directory
HHC00007I Previous message from function 'process_args' at impl.c(1900)
HHC02343S Terminating due to 1 argument errors
HHC00007I Previous message from function 'impl' at impl.c(1146)
HHC01420I Begin Hercules shutdown
HHC01423I Calling termination routines
HHC01424I All termination routines complete
HHC01425I Hercules shutdown complete
HHC01412I Hercules terminated
```

If you specify valid arguments however, it will startup normally in panel mode and the version information is shown in the panel:

![image](https://user-images.githubusercontent.com/1486786/214383580-83d28a02-e08f-4150-bdcb-81fdeed2180a.png)

(in the above, I specified `hercules -f none` to indicate no config file.)

But doing so is inconvenient just to determine Hercules's version, especially since users might not be familiar with that particular feature/option of Hercules or even be familiar with _**any**_ of Hercules's various command line arguments!

This needs to be fixed.

**Comment by Fish-Git** [comment]
Well THAT was easy!

Fixed by commit f7373b7c2ddd29879fbf2992fcd54bae1794fb55.

Closing.

**Comment by wrljet** [comment]
I've been meaning to complain about that for a while.
